tests/processor.test.js

```diff
@@ -345,6 +345,105 @@ describe("processor", () => {
 					assert.strictEqual(blocks[0].filename, "0.js");
 				});
 
+				it("should parse a double-quoted filename from meta", () => {
+					const code =
+						prefix +
+						[
+							'```  js filename="abc.js"',
+							"var answer = 6 * 7;",
+							"```",
+						].join("\n");
+					const blocks = processor.preprocess(code);
+
+					assert.strictEqual(blocks.length, 1);
+					assert.strictEqual(blocks[0].filename, "abc.js");
+				});
+
+				it("should parse a single-quoted filename from meta", () => {
+					const code =
+						prefix +
+						[
+							"```  js filename='abc.js'",
+							"var answer = 6 * 7;",
+							"```",
+						].join("\n");
+					const blocks = processor.preprocess(code);
+
+					assert.strictEqual(blocks.length, 1);
+					assert.strictEqual(blocks[0].filename, "abc.js");
+				});
+
+				it("should parse a double-quoted filename in a directory from meta", () => {
+					const code =
+						prefix +
+						[
+							'```  js filename="abc/def.js"',
+							"var answer = 6 * 7;",
+							"```",
+						].join("\n");
+					const blocks = processor.preprocess(code);
+
+					assert.strictEqual(blocks.length, 1);
+					assert.strictEqual(blocks[0].filename, "abc/def.js");
+				});
+
+				it("should parse a single-quoted filename in a directory from meta", () => {
```

Comment:
Can you add an integration test that uses ESLint to process a file with slashes in the code block filename? 

I *think* it should work okay but I'm not 100% sure if there's some assumption that the filename doesn't have a slash somewhere.

Reply:
Is 235afbc what you're looking for on this end?

Reply:
Yup. :+1: